node/pegnet/addresses.go

```diff
@@ -306,7 +414,8 @@ func (Pegnet) selectBalances(q QueryAble, adr *factom.FAAddress) (map[fat2.PTick
 // balances for the given address. If the address is not in the database,
 // the map will contain 0 for all valid PTickers. This works on the pending tx
 func (p *Pegnet) SelectAllBalances() ([]BalancesPair, error) {
-	rows, err := p.DB.Query(`SELECT * FROM pn_addresses;`)
+	stmt := fmt.Sprintf(`SELECT %s FROM pn_addresses;`, addressSelectCols)
```

Comment:
just a minor cleanup: `stmt` doesn't need to exist

Reply:
I prefer to store them in a variable so if I need to debug I can throw a println and checkout the query. But I can remove the variable if you'd prefer, I'm not super adamant about it.

Reply:
That's good enough to justify keeping it